project/Settings.scala

```diff
@@ -0,0 +1,83 @@
+import sbt._
+
+/**
+ * Application settings and dependencies
+ */
+object Settings {
+  object Definitions {
+    /** The name of the application */
+    val name = "ocs3"
+
+    /** Project version */
+    val version = "2016001.1.1"
```

Comment:
Hmm, I hadn't seen this version alternative.  I wonder if we should abandon the semester in the version number altogether and just count up like normal software products.